README.md

```diff
@@ -1,24 +1,314 @@
 # CSS Fundamentals
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/-k-1TU8qq0Q?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe><p><a href="https://www.youtube.com/watch?v=-k-1TU8qq0Q">CSS Fundamentals</a></p>
+## Problem Statement
 
+You know the basics of CSS and how to use it to style HTML pages. But you want
+to customize your presentation further. Fortunately, CSS has several fundamental
+powers to explore, including selector types, color values and comments.
+
+## Objectives
+
+1. Identify CSS syntax
+2. Identify CSS use formats
+3. Utilize various types of CSS selectors
+4. Utilize various types of color values in CSS
+5. Recognize and implement CSS comments
+
+## Identify CSS Syntax
+
+Before we turn to the various ways we can extend CSS rules, let's go over the
+foundational syntax CSS uses.
+
+<img src="https://curriculum-content.s3.amazonaws.com/fewds/css-syntax.png" alt="CSS syntax diagram labeling selector, property and value" width="500" />
+
+We create a CSS rule by defining the selector, which matches the HTML element we
+want to style. Inside the curly braces we declare the property we want to change
+and, after the colon, we set the value we want to change that property to. We
+end the declaration with a semicolon. Here, we are selecting the `p` element and
+displaying its color as blue.
+
+## Identify CSS Use Formats
+
+There are three distinct formats that you might see CSS appear in: inline,
+internal (or embedded) and external.
+
+Inline includes the styles directly into the HTML element with the `style`
+attribute.
+
+```
+<p style="color: blue;"></p>
+```
+
+While you might see this sort of styling in something like the code for an HTML
+email, this is generally not best practice for two reasons. The primary reason
+is because it only affects that single element. If we want all paragraph
+elements on our page to appear blue, we would have to add that attribute to
+every element individually, which is inefficient and difficult to maintain into
+the future. That brings us to the second reason to avoid inline CSS: it breaks
+our principle of separation of content and presentation.
+
+Internal CSS is inside of `style` tags in the HTML document's `head` section.
+
+```
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      p { color: blue; }
+    </style>
+  </head>
+  <body>
+
+  <p>This is a paragraph.</p>
+
+  </body>
+</html>
+```
+
+This rule will display all paragraphs in this document as blue, which is a step
+up in scope from the inline styles that only apply to single elements. But this
+CSS will only apply to the single document. Other paragraph elements on other
+pages in the same website will be unaffected.
```

Comment:
> "will not know about this change. It would be pretty weird to have the "About Me" page have one appearance and the "My Favorite Poodles" have a different appearance." We want _all_ the pages in our site to have the same look and feel.